bakerydemo/static/css/font-open-sans.css

```diff
@@ -2,6 +2,7 @@
 @font-face {
```

Comment:
there's quite a few versions of open sans being loaded in - I wonder if they're all being used?

Reply:
Made note to raise with Thibaud